modules/policy-criteria.adoc

```diff
@@ -479,6 +479,30 @@ For example `oc`, or `kubectl`.
 | ✓
 | Deploy
 
+| Liveness Probe
+| Whether the container defines a liveness probe.
+| 3.69 and newer
+| ✕
+| ✕
+| ✕
+| Deploy
```

Comment:
Shouldn't this be deploy and build?

Reply:
Tbh I'm a bit unsure what the phase is supposed to mean. Does `Build` refer to the building of a docker image? In that case it would not make sense here. 

A similar criteria would be `Container CPU Request`, which is also just tagged as `Deploy`. In fact there I can't find any `Build & Deploy` tags.